### DISCUSSION TITLE
SMS MFA Sample

### DISCUSSION THREAD

**Discussion by larderino1** [neos-sdi/adfsmfa#156] [Q&A] [answered]
Hello! I have a question about this example, or rather the implementation of the UI part. This version of the MFA release provides the UI or do you need to finalize the rendering of the user interface manually?

**Comment by redhook62** [comment, marked as answer]
Hi,

Yes, this is an example allowing you to use SMS (one way or two way), of course there is an interface in the Web UI part, which will partly take over the elements of the provider that you have to develop.
There is another very educational example -> the Quiz Sample

regards
